src/test/java/com/wikia/webdriver/elements/communities/mobile/components/LightboxComponentObject.java

```diff
@@ -42,22 +42,24 @@ public boolean isCurrentImageVisible() {
     return true;
   }
 
-  public boolean isLightboxHeaderDisplayed() {
-    return !lightboxHeader.getCssValue("display").contains("none");
-  }
-
   public boolean isLightboxFooterDisplayed() {
-    return !lightboxFooter.getCssValue("display").contains("none");
+    try {
+      wait.forElementVisible(lightboxFooter);
+
+      return true;
+    } catch (TimeoutException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)